WcaOnRails/db/migrate/20160610191605_create_competition_tabs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCompetitionTabs < ActiveRecord::Migration
+  def change
+    create_table :competition_tabs do |t|
+      t.string :competition_id
```

Comment:
I don't know about such a list but fixing this (and similar ones) would be as easy as grepping for `belongs_to :competition`.